### PR TITLE
Fix coverage

### PR DESCRIPTION
**Pull request by JSAssassin**
Note: `codecov-action@v1` has been deprecrated https://github.com/codecov/codecov-action#%EF%B8%8F--deprecration-of-v1, so I've updated it to v2.

### CODE REVIEW

File: test/package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
   "scripts": {
     "test": "node --preserve-symlinks test.js test",
     "coverage": "cross-env NODE_ENV=test nyc --reporter=lcov --reporter=text-summary npm test",
-    "coverage-ci": "cross-env NODE_ENV=test nyc --reporter=lcovonly npm test",
+    "coverage-ci": "cross-env NODE_ENV=test nyc --reporter=text-lcov npm test > coverage.lcov",
```
Comment (mattcollier) [inline test/package.json:10]: Is this change required?  I think we may be piping like this elsewhere, but I think there are also examples of a command line option that does not require special pipes.  Might also involve changing the path to the coverage report on L74 of the workflow.

Comment (JSAssassin) [reply test/package.json]: Thanks, that change was not required. I've reverted it back and updated the coverage file path.

**Review by dlongley — APPROVED** [review]
Can I merge this now?

**Review by davidlehn — APPROVED** [review]
